README.md

```diff
@@ -1,2 +1,28 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a): The 'HasDuplicate(list)' function goes compares every member of a list to every member of the same list (including, rather inefficiently, comparing the same member against itself). 
+If i and j are the same, it also asks if it is the same space in the list. If it isn't, the list contains a duplicate entry and we return true.
+
+(b): The running time of the algorithm is quadratic ( O(n^2) ) because, since the algorithm is comparing each element of the list (including the same element), it is going through the the list exactly n^2 times.
+
+(c) The algorithm isn't correct. If a for loop runs from 0 to -1, it will crash. If i = 0, then i-1 will be -1, resulting in a crash. If we assume I'm incorrect and the algorith won't crash, then this new pair of for-loops should avoid comparing each element to itself and comparing two elements more than once.
```

Comment:
By convention, if the end point of a for loop is less than the start point, the loop does nothing. For example the following code in Python prints nothing (and does not crash):
```
for i in range(-1):
   print(i)
```
The rest of your answer is correct.